test/topology_raft_disabled/test_mutation_schema_change.py

```diff
@@ -0,0 +1,11 @@
+
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+"""
+Test repro of failure to store mutation with schema change and a server down
+"""
+from test.topology.test_mutation_schema_change import test_mutation_schema_change, \
```

Comment:
I think running these tests in two suites is unnecessary. The downside is increasing the time to run all tests, in particular CI runs.

Also the tests you made use the `raft_server_snapshot_reduce_threshold` injection, so let's run them only in topology tests only.